finetuner-workflow/finetuner/finetuner.py

```diff
@@ -891,6 +917,8 @@ def evaluate(
 # use for this model and GPU.
```

Comment:
nit - caps

Reply:
That comment is part of a whole sentence, so it does not need to be capitalized. I just checked the file to see if the grammar was off in the comments and it seemed to be fine